research/solver.py

```diff
@@ -40,40 +40,13 @@ def min_lp(lp, *args, M):
     return X
 
 
-# https://or.stackexchange.com/a/712
-@static_vars(var_id=0)
-def max2_lp(lp, a, b, M):
-    X = plp.LpVariable(f'max2_X_{max2_lp.var_id}')
-    y = plp.LpVariable(f'max2_y_{max2_lp.var_id}', cat=plp.LpBinary)
-    max2_lp.var_id += 1
-
-    lp += a - b <= M * y
-    lp += b - a <= M * (1 - y)
-    lp += X >= a
-    lp += X >= b
-    lp += X <= a + M * (1 - y)
-    lp += X <= b + M * y
-
-    return X
-
-
-def max_lp(lp, *args, M):
```

Comment:
why are we removing these? I know they aren't used now but I think it could stay as a useful tool just in case no?

Reply:
Ok, sure. I will bring it back